plugins/bundle/config.go

```diff
@@ -152,7 +154,7 @@ func (c *Config) IsMultiBundle() bool {
 	return c.Name == ""
 }
 
-func (c *Config) validateAndInjectDefaults(services []string, keys map[string]*bundle.KeyConfig) error {
+func (c *Config) validateAndInjectDefaults(services []string, keys map[string]*keys.KeyConfig) error {
```

Comment:
💭 It's kind of tempting to call it `Config` so we can say `keys.Config` here, isn't it? 😄

Reply:
Hmm yeah that's not a bad idea. We still have an alias to the "old" type in the bundle package so this wouldn't break anything if we changed it now.

Reply:
Fixed.